src/libraries/System.Diagnostics.EventLog/tests/System/Diagnostics/Reader/ProviderMetadataTests.cs

```diff
@@ -90,6 +91,7 @@ public void ProviderNameTests(bool noProviderName)
         }
 
         [ConditionalFact(typeof(Helpers), nameof(Helpers.SupportsEventLogs))]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/64153", typeof(PlatformDetection), nameof(PlatformDetection.IsWindows), nameof(PlatformDetection.IsX64Process))]
```

Comment:
>nameof(PlatformDetection.IsWindows), nameof(PlatformDetection.IsX64Process)) [](http://example.com/codeflow?start=98&length=76)

nit: you can combine these 2 conditions in PlatformDetection. Something like `IsX64ProcessOnWindows`

Reply:
Done!